tests/integration/test_monitors.py

```diff
@@ -122,21 +122,20 @@ def test_monitor_post_json_body(mini_sentry, relay):
     relay = relay(mini_sentry)
     mini_sentry.add_basic_project_config(42)
 
-    check_in = generate_check_in("my-monitor")
```

Comment:
`generate_check_in` wasn't giving us much, I just dropped it in favor of inlining the check-in body.